src/PointwiseFunctions/AnalyticSolutions/EinsteinSolutions/Minkowski.hpp

```diff
@@ -78,4 +78,16 @@ class Minkowski {
   // clang-tidy: google-runtime-references
```

Comment:
Commit message should say what you're adding comparison operator too. I think we can in general stop saying we are adding a test for something in the commit message, since that's a basic requirement of getting the code to pass review. I  know Gabe likes to do this, but I think it should be obvious that you did it, otherwise your commit will be rejected. Now, if you make a separate commit that only adds tests, then of course it makes sense to have that as the commit message :)